shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/ui/WillFeedDialog.kt

```diff
@@ -63,6 +66,6 @@ fun WillFeedDialog(onAgreeClick: () -> Unit) {
             )
         }
 
-        else -> {}
+        Dismissed -> {}
```

Comment:
Mb rename to DialogDismissed, not obvious name Dismissed state with Confirmation Requested, Gps Setting Requested and etc.
```suggestion
        DialogDismissed -> {}
```

Reply:
Done